### ISSUE TITLE
Check: Radio elements not grouped semantically with a identical name

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#794]
### Expected behavior

Radio button group has same name attribute value so browser knows they are related as a group to make one selection within. This is the HTML standard for structure to have these attribute values identical to make a radio button group. 
### Actual behavior

This might just be a demo issue, but radio group radiobuttons all have different name attributes so they aren't associated with each other. Screenreaders read each radiobutton as its own element and not related to the group (e.g., radiobutton 1 of 1, not radiobutton 1 of 4)

**Comment by Blackbaud-PatrickOFriel** [comment]
Yeah looks like a demo issue, should be pretty simple to fix

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.3](https://github.com/blackbaud/skyux/releases/tag/v1.7.3)
